source/crd_test.go

```diff
@@ -72,6 +72,7 @@ func startCRDServerToServeTargets(endpoints []*endpoint.Endpoint, apiVersion, ki
 			Name:        name,
 			Namespace:   namespace,
 			Annotations: annotations,
+			Labels:     labels,
```

Comment:
One nit pick with spacing to line up the values
```suggestion
			Labels:      labels,
```